SX1276/SX1276_LoRaRadio.cpp

```diff
@@ -1876,9 +1869,10 @@ void SX1276_LoRaRadio::handle_dio0_irq()
                             -(read_register(REG_RSSIVALUE) >> 1);
 
                     _rf_settings.fsk_packet_handler.afc_value =
-                            (int32_t) (float) (((uint16_t) read_register(REG_AFCMSB) << 8)
+                            (int32_t) (uint64_t) ((((uint16_t) read_register(REG_AFCMSB) << 8)
```

Comment:
Is it more readable as shown below?
```suggestion
 _rf_settings.fsk_packet_handler.afc_value = (int32_t) (uint64_t) (
    (((uint16_t) read_register(REG_AFCMSB) << 8) | (uint16_t) read_register(REG_AFCLSB)) *
    (uint64_t) FREQ_STEP
) / 100000000;
```

Reply:
I rearranged the statement for readability